src/plots/cartesian/layout_defaults.js

```diff
@@ -74,24 +76,24 @@ module.exports = function supplyLayoutDefaults(layoutIn, layoutOut, fullData) {
         if(trace.type === 'funnel') {
             if(trace.orientation === 'h') {
                 if(xaName) xaMayHide[xaName] = true;
-                if(yaName) yaMayBackward[yaName] = true;
+                if(yaName) yaMayReverse[yaName] = true;
             } else {
                 if(yaName) yaMayHide[yaName] = true;
             }
+            yaMustNotScaleanchor[yaName] = true;
+        } else if(trace.type === 'image') {
+            if(yaName) yaMayReverse[yaName] = true;
+            if(yaName) yaMayScaleanchor[yaName] = true;
```

Comment:
We can combine these two `if(yaName)` conditions.

Reply:
Done in d71c3b345d2eecb7b3087c383f00f5f102b9c2f6